client/components/theme/index.jsx

```diff
@@ -233,8 +242,48 @@ export class Theme extends Component {
 		);
 	};
 
+	goToCheckout = () => {
+		const { siteSlug } = this.props;
+		const plan = 'premium';
+
+		if ( siteSlug ) {
+			const params = new URLSearchParams();
+			params.append( 'redirect_to', window.location.href.replace( window.location.origin, '' ) );
+
+			window.location.href = `/checkout/${ encodeURIComponent(
+				siteSlug
+			) }/${ plan }?${ params.toString() }`;
+		}
+	};
+
+	getUpsellMessage() {
+		const { isPremiumThemesAvaiable, theme, price, translate } = this.props;
```

Comment:
Minor spelling error throughout the PR: `Avaiable` -> `Available`

Reply:
Thanks for catching it!